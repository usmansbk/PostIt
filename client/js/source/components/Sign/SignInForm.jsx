import InputField from '../common/InputField.jsx';
import Button from '../common/Button.jsx';
import React from 'react';

export default function SignInForm(props) {
  return (
      <form>

        <InputField className='input-field' id='username' type='text' name='username' label='Username'/>

        <InputField className='input-field' id='password' type='password' name='password' label='Password'>
          <span><a href='#'>Forgot account?</a></span>
        </InputField>
        
        <Button className='center-align' value='Sign In'></Button> 
      </form>
  )
}
