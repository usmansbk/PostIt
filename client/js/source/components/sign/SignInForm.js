import InputField from '../common/InputField';
import Button from '../common/Button';
import React from 'react';

export default () => {
  return (
      <form>

        <InputField className='input-field' id='username' type='text' name='username' label='Username'/>

        <InputField className='input-field' id='password' type='password' name='password' label='Password'>
          <span><a href='#'>Forgot account?</a></span>
        </InputField>

        <Button className='center-align' color='blue' value='Sign In'></Button>
      </form>
  )
}
