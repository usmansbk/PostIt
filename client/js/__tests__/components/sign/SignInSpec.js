import React from 'react';
import SignInForm from '../../../src/components/sign/SignInForm';
import renderer from 'react-test-renderer';

describe('<SignInForm />', () => {
	test('renders sign-in form', () => {
		const component = renderer.create(
			<SignInForm />
		);

		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});