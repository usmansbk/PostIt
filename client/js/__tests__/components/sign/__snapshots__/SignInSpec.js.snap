// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignInForm /> renders sign-in form 1`] = `
<form>
  <div
    className="input-field"
  >
    <label
      htmlFor="username"
    >
      Username
    </label>
    <input
      className="input-field"
      id="username"
      label="Username"
      name="username"
      type="text"
    />
  </div>
  <div
    className="input-field"
  >
    <label
      htmlFor="password"
    >
      Password
    </label>
    <input
      className="input-field"
      id="password"
      label="Password"
      name="password"
      type="password"
    >
      <span>
        <a
          href="#"
        >
          Forgot account?
        </a>
      </span>
    </input>
    <span
      className="helper-text"
    >
      <span>
        <a
          href="#"
        >
          Forgot account?
        </a>
      </span>
    </span>
  </div>
  <div
    className="center-align"
  >
    <button
      className="btn waves-effect waves-light blue"
      type={undefined}
    >
      Sign In
    </button>
  </div>
</form>
`;
